README.rst

```diff
@@ -10,41 +10,65 @@ Some documentation is also available on the Flatpak wiki and as part of the
 The docs are written in `reStructuredText
 <http://www.sphinx-doc.org/rest.html>`_ and contributions are welcome!
 
-Building
---------
+Setup Development
+-----------------
 
 To build the docs locally, first install ``sphinx``, ``furo`` and
 ``sphinx-copybutton``.
-On Fedora this can be with::
+
+On **Fedora** this can be done with::
 
   sudo dnf install python3-sphinx python3-furo python-sphinx-copybutton python-sphinxext-opengraph
 
-On Debian this can be with::
+On **Debian** this can be done with::
 
   sudo apt install python3-sphinx python3-furo python-sphinx-copybutton python-sphinxext-opengraph
 
-Then run ``make html`` in the ``docs`` directory.
+With **pip**, this can be done with::
+
+  pip install -r requirements.txt sphinx-intl
+
+Build the Documentation
+-----------------------
+
+After setup, you can build the documentation::
+
+  make -C docs html
+
+Run the tests::
 
-You can then execute ``cd _build/html && python3 -m http.server`` and follow
-the HTTP link printed by Python to view the docs in your browser.
+  make -C docs linkcheck
+
+You can run an HTTP server and follow the printed link
+(`0.0.0.0:8000 <http://0.0.0.0:8000>`_)
```

Comment:
```suggestion
(`localhost:8000 <http://localhost:8000>`_)
```

Reply:
I see @bbhtt recommended this. But I don't understand why. `0.0.0.0` is a fake address, generally used to refer to listening on a socket to "everything". If something prints it, its because thats what the server is listening on.

A client would connect to its own address, such as `localhost` which is a valid address.

Reply:
Yea, technically it is correct. I suggested 0.0.0.0 because that what the printed link is and to be explicit because 0.0.0.0 is "special".